hid-uclogic-params.c

```diff
@@ -1279,10 +1279,10 @@ static bool uclogic_params_ugee_v2_has_battery(struct hid_device *hdev)
 	 * version.
 	 */
 	if (hdev->vendor == USB_VENDOR_ID_UGEE &&
-	    hdev->product == USB_DEVICE_ID_UGEE_XPPEN_TABLET_DECO_L) {
+	    hdev->product == USB_DEVICE_ID_UGEE_XPPEN_TABLET_DECO_L || hdev->product == USB_DEVICE_ID_UGEE_XPPEN_TABLET_DECO_M) {
```

Comment:
Do you have a Deco MW? This makes sense, but sometimes vendors surprise you and change things.
If this is untested, I'd prefer to not change it. Someone with a Deco MW can always fix and test this in the future.

Reply:
I do indeed have an MW, however, I am not certain of how to test if this device works with this quirk, so I will revert it.

Reply:
If you see the battery level and status (charging/discharging) when you connect your device, it means that no changes are required.

However, if this method returns false for your MW, you'll need to add a condition similar to this one.

In GNOME, the battery is displayed in System settings > Power. Other desktop environments display it in other places.